CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.20.0
```

Comment:
This should probably be a point release: 0.19.1

Reply:
Fixed in 029932a1c073bde2f7f5ce9dfbe32c342852a7eb